test/test_browser.py

```diff
@@ -4355,11 +4355,11 @@ def test_wasm_locate_file(self):
 
   @also_with_threads
   def test_utf8_textdecoder(self):
-    self.btest_exit('benchmark_utf8.cpp', 0, args=['--embed-file', test_file('utf8_corpus.txt') + '@/utf8_corpus.txt', '-sEXPORTED_RUNTIME_METHODS=[UTF8ToString]'])
+    self.btest_exit('benchmark', 'benchmark_utf8.cpp', 0, args=['--embed-file', test_file('utf8_corpus.txt') + '@/utf8_corpus.txt', '-sEXPORTED_RUNTIME_METHODS=[UTF8ToString]'])
```

Comment:
Use `/` here and in test_core.py too.

Reply:
Ops, silly me, I was looking at a quick glance that there was only one place where that occurred.. will update.